src/EditorFeatures/CSharpTest/Completion/CompletionProviders/DeclarationNameCompletionProviderTests.cs

```diff
@@ -1569,6 +1573,7 @@ class Test
             await VerifyItemExistsAsync(markup, "cancellationTokens");
             await VerifyItemExistsAsync(markup, "cancellations");
             await VerifyItemExistsAsync(markup, "tokens");
+            await VerifyItemExistsAsync(markup, "Enumerable");
```

Comment:
Should `Enumerable` be in camelCase here (and other such tests), considering, that all other suggestions are in it?

Reply:
Yes, it should. Actually, both Pascal case and camel case should exist, since the definition could be either a field or a property at this point.

Reply:
tbh... is this really desirable?  the other names seem quite good.